hbase-assembly/pom.xml

```diff
@@ -373,5 +373,14 @@
       <artifactId>opentelemetry-javaagent</artifactId>
       <classifier>all</classifier>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>compile</scope>
```

Comment:
This works? Great!

Because `compile` is the default scope, someone later may be tempted to remove this distinction. Maybe add a comment to future do-gooders?

Reply:
Done.